src/main/java/com/google/api/codegen/transformer/go/GoSurfaceNamer.java

```diff
@@ -100,9 +100,31 @@ public String getLongRunningOperationTypeName(ModelTypeTable typeTable, TypeRef
 
   @Override
   public List<String> getDocLines(Method method, GapicMethodConfig methodConfig) {
-    String text = DocumentationUtil.getDescription(method);
-    text = lowerFirstLetter(text);
-    return super.getDocLines(getApiMethodName(method, methodConfig.getVisibility()) + " " + text);
+    return super.getDocLines(
+        putDocMethodName(
+            method.getSimpleName(),
+            DocumentationUtil.getDescription(method),
+            methodConfig.getVisibility()));
+  }
+
+  /**
+   * Docs in Go usually starts with the name of the method. Prepend the method name if the name
+   * isn't already the first word.
+   */
+  @VisibleForTesting
+  String putDocMethodName(String methodName, String doc, VisibilityConfig visibility) {
+    doc = doc.trim();
+    String firstWord = doc.trim();
```

Comment:
I'm confused by this. According to https://docs.oracle.com/javase/7/docs/api/index.html?java/lang/String.html, trim just removes whitespace, so calling it a second time won't do anything. 

Maybe just declare firstWord, and loop over doc below?